helm/hpcc/templates/esp.yaml

```diff
@@ -1,5 +1,29 @@
+{{/*
+Esp configmap
+Pass in dict with root and me
+*/}}
+{{- define "hpcc.espConfigMap" }}
+apiVersion: v1
+metadata:
+  name: {{ .me.name }}-configmap
+data:
+  {{ .me.name }}.yaml: |
+    version: 1.0
+    esp:
+{{ toYaml (omit .me "logging") | indent 6 }}
+{{- include "hpcc.generateLoggingConfig" (dict "root" .root "me" .me) | indent 6 }}
+      queues:
+{{ include "hpcc.generateConfigMapQueues" .root | indent 6 }}
+      services:
+{{ include "hpcc.generateConfigMapServices" .root | indent 6 }}
+{{ include "hpcc.generateVaultConfig" (dict "root" .root "categories" (list "storage" "esp" ) ) | indent 6 }}
+    global:
+{{ include "hpcc.generateGlobalConfigMap" .root | indent 6 }}
+{{- end -}}
+
 {{ range $.Values.esp -}}
 {{- if not .disabled -}}
+{{- $configSHA := include "hpcc.espConfigMap" (dict "root" $ "me" .) | sha256sum }}
```

Comment:
I wonder whether a $scope "= dict "root" $ "me" . at the top of some of these would be helpful, as it's passed to a LOT of templates